aten/src/ATen/native/native_functions.yaml

```diff
@@ -12609,6 +12609,17 @@
 - func: linalg_multi_dot.out(Tensor[] tensors, *, Tensor(a!) out) -> Tensor(a!)
   python_module: linalg
 
+## Functions related to the `torch.nested` namespace
+# Note [nested namespace binding]
+# Functions in the nested python module should have their names start with
+#   "nested_" underscore and be bound to the desired Python name in
+#   torch/nested/__init__.py, and the desired C++ name in torch/csrc/api/include/torch/nested.h.
+#   The "nested_" names should be hidden from the user and not documented.
+
+- func: nested_to_padded_tensor(Tensor self, float padding, int[]? output_size=None) -> Tensor
```

Comment:
Is it the pattern to create aliases for namespaced functions? I think it makes more sense to only have one function. If the pattern is to create aliases for namespace functions then then we should create a name neseted_aliases.cpp and move the definitions there.